NYPLAudiobookToolkit/Core/AudiobookBookmarks.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+public protocol AudiobookBookmarksBusinessLogicDelegate {
+  var bookmarksCount: Int { get }
+  
+  func bookmark(at index: Int) -> NYPLAudiobookBookmark?
+  func addAudiobookBookmark(_ chapterLocation: ChapterLocation)
+  func deleteAudiobookBookmark(at index: Int)
+  func syncBookmarks(completion: @escaping (_ success: Bool) -> ())
+}
+
+public final class NYPLAudiobookBookmark {
```

Comment:
can this be a struct? especially since it looks immutable, and a bookmark does seem like a value type. Or do we need the class semantics for some reason?

Reply:
Swift's `final` can only be used on class but not struct. I am fine with not using `final` since this repo is only used by us.

Reply:
Actually, `annotationId` might change after we complete upload the annotation to server. I will be making more changes to this class once I start implementing the business logic. I will leave this unchanged with a comment for now and we can revise this when it's ready.